src/D2L.Bmx.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
please put the project in a folder with the same name.
i.e. files should look like
```
src/D2L.Bmx/D2L.Bmx.csproj
src/D2L.Bmx/*.cs
```